ArjunaJTA/jdbc/tests/classes/com/hp/mwtests/ts/jdbc/basic/ConnectionImpleTest.java

```diff
@@ -125,13 +124,14 @@ public void beforeCompletion() { }
             public void afterCompletion(int status) {
                 try {
                     connectionToTest.isClosed();
+                    connectionToTest.close();
```

Comment:
I updated the message. 
RE connection.close() I am not sure if you are saying it doesn't matter because we are using a mock in which case I would disagree because someone might change it from a mock to a real connection in the future and then it would break. OTOH aybe you are saying even with a real connection you would not need the .close in which case I can remove it.
I do see from the rest of your tests that you do have the .close()